tests/unit/restapi/v1/test_queue.py

```diff
@@ -0,0 +1,516 @@
+# This Software (Dioptra) is being made available as a public service by the
+# National Institute of Standards and Technology (NIST), an Agency of the United
+# States Department of Commerce. This software was developed in part by employees of
+# NIST and in part by NIST contractors. Copyright in portions of this software that
+# were developed by NIST contractors has been licensed or assigned to NIST. Pursuant
+# to Title 17 United States Code Section 105, works of NIST employees are not
+# subject to copyright protection in the United States. However, NIST may hold
+# international copyright in software created by its employees and domestic
+# copyright (or licensing rights) in portions of software that were assigned or
+# licensed to NIST. To the extent that NIST holds copyright in this software, it is
+# being made available under the Creative Commons Attribution 4.0 International
+# license (CC BY 4.0). The disclaimers of the CC BY 4.0 license apply to all parts
+# of the software developed or licensed by NIST.
+#
+# ACCESS THE FULL CC BY 4.0 LICENSE HERE:
+# https://creativecommons.org/licenses/by/4.0/legalcode
+"""Test suite for queue operations.
+
+This module contains a set of tests that validate the CRUD operations and additional
+functionalities for the queue entity. The tests ensure that the queues can be
+registered, renamed, deleted, and locked/unlocked as expected through the REST API.
+"""
+from __future__ import annotations
+
+from typing import Any
+
+import pytest
+from flask.testing import FlaskClient
+from flask_sqlalchemy import SQLAlchemy
+from werkzeug.test import TestResponse
+
+from dioptra.restapi.routes import V1_QUEUES_ROUTE, V1_ROOT
+
+from ..lib.actions import register_queue
+from ..lib.helpers import is_iso_format
```

Comment:
The number of actions and helpers might grow large as we build out the testing infrastructure for v1. I suggest we export the `actions` and `helpers` module in `tests/unit/restapi/lib/__init__.py` so that we can just import as follows:

```suggestion
from ..lib import actions, helpers
```

Then we can make calls like `actions.register_queue` and `helpers.is_iso_format`, which make the purpose of the functions even more clear (these are "actions", these are "helpers").